nh_eobs_mobile/tests/nh_clinical_form_description/test_to_dict.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
```

Comment:
Could be `Model` model not loaded by Odoo yet, needs to be in module's main `__init__`.